app/models/game.rb

```diff
@@ -30,4 +32,10 @@ class Game < ActiveRecord::Base
   def complexity_name
     COMPLEXITIES[complexity]
   end
+
+  def update_average
+    s = reviews.sum(:game_rating)
+    c = reviews.count
+    update_attribute(:average, c == 0 ? 0.0 : s / c.to_f)
```

Comment:
So you want to calculate an average every time there's a new review? That seems pretty inefficient.